{"name": "Проверка авторизоваться под несуществующим пользователем, запрос возвращает ошибку", "status": "passed", "description": "Запрос на логин с неверным логином и паролем", "start": 1730171469666, "stop": 1730171470020, "uuid": "b933cff4-2cfe-42ba-b1a4-f4065e09b216", "historyId": "71fdaa70d618b15810f59d2632c03267", "testCaseId": "71fdaa70d618b15810f59d2632c03267", "fullName": "tests.test_login_user.TestLoginUser#test_user_login_wrong_login_password_failed", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "MacBook-Pro-user.local"}, {"name": "thread", "value": "1227-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}